### PR TITLE
Add support for mapzen

### PR DESCRIPTION
**Pull request by andrewlively**

### CODE REVIEW

**Comment by nchaulet** [comment]
Thanks for the PR could you please fix the test before I merge it https://travis-ci.org/nchaulet/node-geocoder/jobs/233713544

File: test/geocoder/mapzengeocoder.js [hunk 1]
```diff
@@ -0,0 +1,106 @@
+(function() {
+    var chai = require('chai'),
+        should = chai.should(),
+        expect = chai.expect,
+        sinon = require('sinon');
+
+    var MapzenGeocoder = require('../../lib/geocoder/MapzenGeocoder.js');
```
Comment (nchaulet) [inline test/geocoder/mapzengeocoder.js]: `../../lib/geocoder/MapzenGeocoder.js` => `../../lib/geocoder/mapzengeocoder.js`

**Comment by andrewlively** [comment]
Thanks for catching that! I have pushed this change

**Comment by nchaulet** [comment]
v3.17.0 thanks for the PR

**Pull request merged**